terraform/modules/configs/deploy-all/variables.tf

```diff
@@ -6,48 +6,102 @@ variable "environment" {
   type = string
 }
 
-variable "stage" {
+variable "az_names" {
+  type    = list(string)
+  default = ["eu-west-2a", "eu-west-2b"]
+}
+
+#######################
+# SPREE SHARED
+#######################
+#/bar/{env}-basic-auth-enabled - flag to enable/disable basic auth
+variable "basic_auth_enabled" {
+  type    = bool
+  default = true
+}
+
+#######################
+# SPREE CLIENT
+#######################
+variable "ecr_image_id_client" {
   type    = string
-  default = "staging"
+  default = "latest"
 }
 
-variable "api_rate_limit" {
+variable "client_ec2_instance_type" {
+  type    = string
+  default = "t2.medium"
+}
+
+variable "client_cpu" {
   type    = number
-  default = 10000
+  default = 2048
 }
 
-variable "api_burst_limit" {
+variable "client_memory" {
   type    = number
-  default = 5000
+  default = 4096
 }
 
-variable "ecr_image_id_spree" {
+# (used to be param: /bat/{env}-documents-terms-and-conditions-url)
+variable "documents_terms_and_conditions_url" {
   type    = string
-  default = "latest"
+  default = "https://www.crowncommercial.gov.uk/agreements/RM6147"
 }
 
-variable "ecr_image_id_client" {
-  type    = string
-  default = "latest"
+# (used to be param: /bat/{env}-enable-quotes)
+variable "enable_quotes" {
+  type    = bool
+  default = true
 }
 
-variable "ecr_image_id_s3_virus_scan" {
+# (used to be param: /bat/{env}-enable-basket)
+variable "enable_basket" {
+  type    = bool
+  default = true
+}
+
+# (used to be param: /bat/{env}-sidekiq-concurrency)
+variable "sidekiq_concurrency" {
+  type    = number
+  default = 100
+}
+
+# (used to be param: /bat/{env}-sidekiq-concurrency-searchkick)
+variable "sidekiq_concurrency_searchkick" {
+  type    = number
+  default = 40
+}
+
+# (used to be param: /bat/{env}-logit-application)
+variable "logit_application" {
   type    = string
-  default = "latest"
+  default = null # this is purposfully null as we do a null check in the main.tf
 }
 
-variable "rollbar_env" {
+# (used to be env var: PAPERTRAIL_HOSTNAME from client.env)
+variable "papertrail_hostname" {
```

Comment:
this papertrail envs can be removed,  as we now using logit.io for logging

Reply:
Great - the less the merrier! I'll remove all those thanks